Dockerfile

```diff
@@ -82,3 +85,8 @@ RUN pip3 install gmsaas
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 RUN gmsaas config set android-sdk-path $ANDROID_HOME
+
+# Flutter
+RUN wget https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_2.0.6-stable.tar.xz
+RUN tar xf flutter_linux_2.0.6-stable.tar.xz
+RUN mv flutter /usr/local/
```

Comment:
@oxtoacart If you want to avoid hard-coding a specific Flutter version here, you could just always obtain the latest version from master (and simplify the above to a one-liner): 

```
RUN git clone https://github.com/flutter/flutter.git /usr/local/flutter
```

Maybe also verify the flutter installation afterwards by running the following?

```
RUN flutter channel master
RUN flutter upgrade
RUN flutter doctor -v
```

Reply:
Oh, you also need this line:

```Dockerfile
ENV PATH $PATH:/usr/local/flutter/bin/:/usr/local/flutter/bin/cache/dart-sdk/bin
```

Reply:
I like running flutter doctor, I'll add that. I'm intentionally pinning flutter 2.0.6, as some of our libraries ran into issues with 2.2.0.